adoc/TRD-wordpress-on-lamp-on-sles.adoc

```diff
@@ -192,7 +192,7 @@ DROP USER ''@localhost;
 ----
 +
 
-See an example below:
+For example:
```

Comment:
We work with complete sentences. "For example:" as it is used here is not a complete sentence. Thus I had used "See an example below". We can also change that to "Find a concrete/practicable example below/here". Of course I am open to better suggestions. But we need an entire sentence here.

```suggestion
Find a practicable example here:
```